lib/src/wrappers/ResponsiveWrapper.tsx

```diff
@@ -1,27 +1,27 @@
 import * as React from 'react';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
-import { useTheme } from '@material-ui/core/styles';
+import { IS_TOUCH_DEVICE_MEDIA } from '../constants/dimensions';
 import { MobileWrapperProps, MobileWrapper } from './MobileWrapper';
 import { DesktopWrapperProps, DesktopWrapper } from './DesktopWrapper';
-import { Breakpoint } from '@material-ui/core/styles/createBreakpoints';
 import { DesktopPopperWrapperProps, DesktopPopperWrapper } from './DesktopPopperWrapper';
 
 export interface ResponsiveWrapperProps
   extends DesktopWrapperProps,
     DesktopPopperWrapperProps,
     MobileWrapperProps {
-  /** Breakpoint when `Desktop` mode will be changed to `Mobile`
-   * @default 'md'
+  /** Css media query when `Mobile` mode will be changed to `Desktop`
+   * @default "@media (pointer: fine)"
+   * @example "@media (min-width: 720px)" or theme.breakpoints.up("sm")
```

Comment:
What do you think of this variation to better match the style of the mono-repository for the description of the props?

```suggestion
  /**
   * CSS media query when "mobile" mode will be changed to "desktop".
   * 
   * @default "@media (pointer: fine)"
   * @example "@media (min-width: 720px)" or theme.breakpoints.up("sm")
```

Reply:
`Mobile` and `Desktop` are used as prefixes for specific components (e.g. `MobileDatePicker`) so it should be more clear for users. Isn't it?

Reply:
`Mobile` and `Desktop` are used as prefixes for specific components (e.g. `MobileDatePicker`) so it should be more clear for users. Isn't it?

Reply:
Using `Mobile` feels like it's a piece of code that can be used on its own. What about 1. `MobileDatePicker` or 2. "mobile", I think that we would lose value if we try to compromise with these two options.